src/login/login.service.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { LoginUserDto } from "./dto/login-user.dto";
+import { InjectModel } from "@nestjs/mongoose";
+import { Model } from "mongoose";
+import { Registration, RegistrationDocument } from "../schemas/registration.schema";
+const jwt = require('jsonwebtoken');
+
+const bcrypt = require('bcryptjs');
+
+@Injectable()
+export class LoginService {
+
+    constructor(@InjectModel(Registration.name) private regModel: Model<RegistrationDocument>) {
+    }
+
+    async loginUser(regDto: LoginUserDto) {
+
+        let {email, password} = regDto;
+        const user = await this.regModel.findOne({email});
```

Comment:
Man, cypher should find it? He knows exactly where it is!!!

Reply:
molodec